apps/rush-lib/src/data/commandLineConfiguration.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
>"$schema": "http://json-schema.org/draft-04/schema#", [](start = 2, length = 53)

Check that this validates, since JsonSchema isn't doing that currently

Reply:
this should be built into node-core-library

---
In reply to: [146732552](https://github.com/Microsoft/web-build-tools/pull/387#discussion_r146732552) [](ancestors = 146732552)

Reply:
out of scope

---
In reply to: [147496188](https://github.com/Microsoft/web-build-tools/pull/387#discussion_r147496188) [](ancestors = 147496188,146732552)